agent/bench-scripts/postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import sys, json, os, logging, shutil
+
+benchmark_run_dir = sys.argv[1]
+benchmark_duration = int(sys.argv[2])
+
+with open('%s/user-benchmark-summary.json' % benchmark_run_dir, 'a+') as data_file:
+    try:
+        data = json.load(data_file)
+    except ValueError as e:
+        print e
```

Comment:
Could we consider using `logging.exception` here, adding a message containing the output of the `tell()` from the file?